src/nanocurrency/accounts.py

```diff
@@ -345,6 +344,6 @@ def generate_seed():
     :return: Seed as a 64-character hex string
     :rtype: str
     """
-    gen = random.SystemRandom()
-    return "".join([gen.choice("0123456789abcdef") for _ in range(0, 64)])
-
+    gen = random.SystemRandom().getrandbits(256)
+    seed = hex(gen).upper()[2:]
```

Comment:
This will sometimes generate seeds that are *not* 64 characters long, and you need to zero pad the seed if this is the case:
```
import random
gen = random.SystemRandom()
while True:
    seed = hex(gen.getrandbits(256)).upper()[2:]
    assert len(seed) == 64, "Length was {}".format(len(seed))
```

Besides that, I don't see why this function would need to be changed?